linkerd/app/outbound/src/http/concrete.rs

```diff
@@ -23,8 +23,14 @@ use tracing::info_span;
 pub enum Dispatch {
     Balance(NameAddr, balance::EwmaConfig),
     Forward(Remote<ServerAddr>, Metadata),
+    Null,
 }
 
+/// A backend dispatcher was null.
+#[derive(Debug, thiserror::Error)]
+#[error("no backend dispatcher configured")]
+pub struct NullDispatcher(());
```

Comment:
It could be. The inner `()` field prevents it from being constructed outside this module. Which, in this case, doesn't really matter --- it's just a bit of a habit from what I would do if this was a public library API.